### ISSUE TITLE
Some Changes

### ISSUE THREAD

**Issue by Ben7230** [gtimelog/gtimelog#223]
First question, is this still being actively developed/maintained? 

First, I like the simple concept of keeping track of time, However the thing that goes against my brain is the fact that you need to enter activity after you finished. I understand the concept, but to me it would make much more sense to type the activity that you want to start and then when you are finished type done or finished. 

To me the logic just flows better. :)

For example When I arrive at work or start working I would type; Arrived or Started

The timer would start

One hour later I go for breakfast so I would type; Breakfast**

The timer would stop

When finished with breakfast type; Work

The timer would start again.

and so on till the end of the day or till I finished my job, and then I would type Done/Going Home**. 

The timer would stop for the day.

To make it easier yet I would make two buttons for each task "Start" and "Finished"  

If I click on the Breakfast start button before I finished the the task that was running it would stop the timer for that task, then after I clicked the Breakfast Finished button, the timer would start again for the same task that was running before. 

Just some thoughts on making this easier to use.

**Comment by jpsnyder** [comment]
While other timekeeping programs work like that (start/stop timer), I think what you describe go against the philosophy of the program. IMHO I like the current method because:
A) Somethings you're not quite sure what you're going to be doing until after you do it. Easier to just log what you did after the fact. 
B) Doing the reverse method requires you to ensure you both properly "start" the day with `arrived **` and "end" the day with `done **`. This can be dangerous if you forget to end the day and the next day you have recorded a 12 hour task :)  The current method just requires you to start the day. And you can literally just stop logging whenever you want.

Obviously this is all about personal preference, but I think there are plenty of other programs that have the start/stop timer technique that one can choose from, I would like to keep this current method for gtimelog, because you can't really get this anywhere else.

**Comment by Ben7230** [comment]
Fair enough, I knew I was going against the philosophy of this program, And it actually works good for what is was designed to do.

I am going to try it and see how it works for me. 

Maybe we could have it as an option. That way the person could select to use it the way it works best for them. 

I have been searching the internet for a program like this and I would say that 99% of them require a online account which I don't need and don't want. 

I work as a subcontractor for someone and I need a easy way to keep track of my hours and also the breaks that I take. I don't have to keep track of multiple task, just 1, Work :)

**Comment by mgedmin** [comment]
> First question, is this still being actively developed/maintained?

Actively developed -- not really; maintained -- yes, for some value of maintained.  I still use gtimelog every day at work, so I'm committed to keeping it working.

I'm utterly failing at my self-imposed goal of building a flatpak package and uploading it to Flathub.  The building bit is done, the uploading is where my undiagnosed ADHD interferes and I just can't make myself get it done.
